pkg/grpc/device_definition.proto

```diff
@@ -250,6 +250,7 @@ message GetDeviceCompatibilityListRequest {
 message Feature {
 	string key = 1;
 	int32 support_level = 2;
+	string css_icon = 3;
```

Comment:
One thing to consider is that the CSS icon is optional in the database. Should be optional in the message here? We could stick with empty, just a question.

Reply:
would that be an optional using something like this? https://pkg.go.dev/github.com/gogo/protobuf/gogoproto#hdr-More_Canonical_Go_Structures

Reply:
Can we leave a comment, at least? "May be blank"?